Added info about some more emulators and polished a bit

SuperGamer18 opened the pull request:
Affected file: emulator-support-and-issues.md
So, as the title says, I added info on a few more emulators and generally polished it. What exactly have I changed? (Yes, I know you could check the history for comparing versions but I'll list the stuff I've changed here anyways)
Also keep in mind I may have missed some things. If so, please let me know!
- Replaced all instances of underscore (_) with asterisk (*) for consistency reasons
- Added 64/ before the 128 in Enterprise 128
- Added the fact that the games played in the gw libretro core are recreations
- Added extra info about how to use RetroAchievements in Dolphin
- Properly capitalized/fixed some of the emulator names
- Added extra standalone emulators and info about them such as Citra, PICO-8, Infuse, etc.
- Added info about DeSmuME not supporting DSi
- Added info about what NetherSX2 is (a mod of AetherSX2)
- Extra emphasis on "recommended" in the Sega Saturn section
- Removed an exclamation mark (!) in the Snes9x 2010 part
- Added info about VeMUlator not supporting savestates
- Marked the libretro core version of Dolphin as not supported (❌) for achievements (applies both to GameCube and Wii)
And that should be it, I think.

SuperGamer18 committed:
Added info about some more emulators and polished a bit

SuperGamer18 committed:
Merge branch 'main' into patch-1

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -1,6 +1,6 @@
-title: Emulator Support
-description: Comprehensive guide to emulator support and compatibility for RetroAchievements, including console-specific issues and recommendations for various cores and emulators.
+Title: Emulator Support
+Description: Comprehensive guide to emulator support and compatibility for RetroAchievements, including console-specific issues and recommendations for various cores and emulators.
```
This is not [conventional frontmatter syntax](https://vitepress.dev/guide/frontmatter). Please revert.

wescopeland commented on docs/general/emulator-support-and-issues.md:
Please undo all of the underscore to star changes.

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -326,7 +328,9 @@ description: Comprehensive guide to emulator support and compatibility for Retro
-- ❌ _Not supported_ - needs hashing method and memory map
+- ❌ *Not supported* - needs hashing method and memory map
+- ❌ Standalone emulator: **Citra**
+  - No longer in active development due to Nintendo making the original creators take it down along with Yuzu. However, an active fork exists, made by PabloMK7 of CTGP-7 fame.
```
We shouldn't wade into the business of commenting on these matters. I really don't want us getting search traffic from these keywords. Please revert.

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -348,28 +352,27 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 
 ### Nintendo DSi
 
-- ✅ libretro core: **melonDS DS**
```
melonDS DS is a core we support. Please revert.

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -419,18 +422,20 @@ description: Comprehensive guide to emulator support and compatibility for Retro
-- ❌ _Not supported_ - needs hashing method and memory map
+- ❌ *Not supported* - needs hashing method and memory map
+- ❌ Standalone emulator: **PICO-8**
+  - Closed source and paid (this is the official way to play PICO-8 games)
```
I don't think we should add this to our docs.

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -456,6 +461,7 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 - ❌ Standalone emulator: **AetherSX2**
   - No longer being developed, has known incompatibilities
 - ❌ Standalone emulator: **NetherSX2**
+  - Mod of the above emulator
```
This should read "Fork of AetherSX2, known to have many issues with achievements and hardcore mode".

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -659,25 +665,29 @@ description: Comprehensive guide to emulator support and compatibility for Retro
+- ❌ *Not supported* - needs hashing method and memory map
+- ❌ Standalone emulator: Infuse
+  - Currently closed source
+  - WIP
```
I think we should remove this.

SuperGamer18 commented:
Alright, I got everything fixed and since I had the time to do it, I added and fixed some other stuff ;3
Here's the new file: 
[emulator-support-and-issues.md](https://github.com/user-attachments/files/16445375/emulator-support-and-issues.md)
(I was too lazy to make another commit in my fork or whatever, this is the first time I ever edited the docs page as well as the first time I ever made a pull request and a fork, I wasn't aware about some of this stuff as I'm not a GitHub nerd)

wescopeland commented:
> I was too lazy to make another commit in my fork or whatever

Please realize that while you may be "too lazy" to make another commit to the branch, you're asking me to expend quite a bit of time to review a raw markdown file. This really isn't going to work.

If you need help getting set up or acclimated with Git, I'm sure someone on QAM or the Writing Team would be more than happy to assist.

SuperGamer18 committed:
Update emulator-support-and-issues.md

Fixed version requested by @wescopeland

SuperGamer18 committed:
Update emulator-support-and-issues.md

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -170,6 +170,7 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 - ❌ _Not supported_ - needs hashing method and memory map
 - ❌ libretro core: **gw**
   - Does not expose memory
+  - Uses recreations (by MADrigal) instead of actual Game & Watch ROMs
```
We can probably remove this too - this seems like a libretro implementation detail.

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -311,16 +313,16 @@ description: Comprehensive guide to emulator support and compatibility for Retro
+### NES/Famicom
 
-- ✅ Standalone emulator: **[RANes](https://retroachievements.org/download.php#ranes)**
+- ✅ Standalone emulator: **[RANes](https:retroachievements.org/download.php#ranes)**
```
This link is now broken.

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -347,7 +349,6 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 - Limited microphone support
 - ✅ libretro core: **DeSmuME**
 - ✅ libretro core: **DeSmuME 2015**
-- ✅ libretro core: **melonDS**
```
[We support the melonDS libretro core](https://docs.libretro.com/guides/retroachievements/#nintendo-ds). Please revert.

SuperGamer18 commented:
https://github.com/SuperGamer18/docs/blob/patch-1/docs/general/emulator-support-and-issues.md I finally got it, the real issue was that the device I originally made these changes with was very underpowered. Sorry for what happened here, I'll try to be more mindful of these kinds of situations next time (I hope this is reviewable)

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -537,7 +539,7 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 ### Sharp X1
 
-- ❌ _Not supported_ - needs hashing method and memory map
+- ❌ _Not supported_- needs hashing method and memory map
```
Let's re-add the whitespace before the dash.

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -685,4 +693,4 @@ description: Comprehensive guide to emulator support and compatibility for Retro
   - Testing
 
-More details on BizHawk cores can be found [here](https://tasvideos.org/BizHawk).
+More details on BizHawk cores can be found [here.](https://tasvideos.org/BizHawk)
```
It's more proper to exclude the punctuation from the hyperlink label.

wescopeland reviewed (commented):
This is reviewable. Thank you for the follow-up & your understanding.

SuperGamer18 committed:
Update emulator-support-and-issues.md

Fixed yet again

SuperGamer18 commented:
https://github.com/SuperGamer18/docs/blob/patch-1/docs/general/emulator-support-and-issues.md Fixed, yet again. Two things I wanna note though:
1: I may have accidentally done some of these mistakes when editing and I didn't realize them. Sorry.
2: I'm kinda confused about the MelonDS thing because for the DSi section you asked me to keep the extra DS in "MelonDS DS" but in the regular DS section you wanted me to remove it.

wescopeland commented:
1. No worries, that is why we have docs reviews. This is one of the most-visited pages under all of `retroachievements.org`, so it's important that the content is heavily vetted. Overall, what we've whittled down to is an improvement. Thank you!

2. "melonDS" and "melonDS DS" are two different libretro cores, both of which are supported.

SuperGamer18 commented:
1: You're welcome, man! To be honest, it isn't hard to figure out why this is one of the most visited pages on the docs and the website in general, it's literally linked right at the home page of `docs.retroachievements.org`! (And I'm pretty sure many users have issues in choosing which core should they use to get those cheevos)
2: I'm a mobile user, so I'm not aware of most libretro cores outside of the ones offered on Android (OS in which I use RetroArch)

wescopeland commented:
@SuperGamer18 Not sure what happened - I was tied up with work and wasn't able to give final approval. If you'd like to reopen the PR I'm happy to walk through getting it merged.

SuperGamer18 commented:
Damn, just as I was fixing the Spanish translation. Oh well, guess I'll take my time to improve the English version even further and update the Spanish version to be on par with the former. BTW should I have both in the same PR or should I open separate PRs for each language?
